### ISSUE TITLE
Documentation for using AUX with Angular it's obsolete

### ISSUE THREAD

**Issue by Leonardonline** [DeutscheSoft/aux-widgets#4]
Could you please provide a documetation updated for use with Angular project?

The [online doc](https://docs.deuso.de/AUX/tutorial-Angular.html) seems to be not updated and not valid more, for example `node_modules/@deuso/aux/angular` do not exists anymore.

**Comment by arneg** [comment]
Yes, that is probably correct. Angular has evolved quite a bit since the corresponding integration was written. I think it might be best to either use the web-components from angular or create angular components and instantiate widgets inside of them in javascript/typescript.

What angular version do you use?

**Comment by Leonardonline** [comment]
We're currently using Angular version 18. 
Would it be possible for you to provide an example (StackBlitz?) demonstrating how to integrate the web components or instantiate the widgets within Angular components? That would be extremely helpful for us to move forward.
